flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -881,24 +883,21 @@ class GroupWindowedTable(
     * }}}
     */
   def select(fields: Expression*): Table = {
```

Comment:
Not sure, I would like to avoid a solution where projections for aggregates are injected at different locations in the code, i.e., injecting a projection before validation for non-windowed aggregates and after validation (or during optimization) for windowed aggregates. I think it would be better to have this consistent.